docs/csi-5to6-upgradepath.md

```diff
@@ -9,13 +9,15 @@ The upgrade path is available in PSO 6.1.x.
 
 ## Restrictions
 1. Upgrade tool will **NOT** be migrating snapshots or snapshots content from PSO 5.x to PSO 6. PSO 6 is currently leveraging beta version of K8s snapshot API and backward compatibility is not guranteed by CSI standard. That means a snapshot created in PSO 5.x will not be used to restore a volume in PSO 6 after the upgrade. 
-2. In addition, the newly installed PSO 6 will need to be in the same Pure namespaces as PSO 5.x. Specifically, that means your `namespace` field in PSO 5.x helm chart `values.yaml` must be exactly the same as your `clusterID`field in PSO 6 `values.yaml` (`namespace` field in PSO 5.x has been renamed to `clusterID` in PSO 6, it is used as prefix in PSO volume names). Note that certain restrictions were imposed on PSO 6 clusterID, if your PSO 5.x namespace does not meet PSO 6 clusterID restrictions, this upgrade tool will not work for you, please reach out for alternatives. 
+2. In addition, the newly installed PSO 6 must be in the same Pure namespaces as PSO 5.x. Specifically, that means your `namespace` -> `pure` field in PSO 5.x helm chart `values.yaml` must be exactly the same as your `clusterID`field in PSO 6 `values.yaml` (`namespace` field in PSO 5.x has been renamed to `clusterID` in PSO 6, it is used as prefix in PSO volume names). Note that certain restrictions were imposed on PSO 6 clusterID, if your PSO 5.x namespace does not meet PSO 6 clusterID restrictions, this upgrade tool will not work for you, please reach out for alternatives. 
 3. If your cluster has more than 500 PVCs/Attachments, it is possible the time taking to construct database content exceed the restart threshold of the controller pods, reach out to PSO team if you run into this situation.
 
 ## Instruction:
-1. prepare your `values.yaml` first for PSO 6 using the PSO 5.x `values.yaml`. They are formatted differently, so you won't be able to simply copy PSO 5.x `values.yaml` over. Schema validation will also help to make sure you comply with PSO 6 formats. 
-2. For upgrade field in PSO 6, it is by default set to `false`. PSO will not start the upgrade if the field is omitted or left to `false`. Set it to `true` to enable upgrade. The upgrade takes place during the installation. 
-3. Install PSO 6 following the instruction provided [here](../pure-pso/README.md). Monitor PSO pods and if all pods are up and running, the upgrade was successful. 
+1. Uninstall PSO 5 following the instruction provided [here](../pure-pso/README.md).  
```

Comment:
This needs to be flagged as an Important item but also state that existing PVCs will NOT be lost due to this step.